### DISCUSSION TITLE
Struggles with chapter creation in docx book project

### DISCUSSION THREAD

**Discussion by iusgit** [quarto-dev/quarto-cli#2512] [Q&A]
Hi,

first thanks for this wonderful tool!

I created a book project and one output format is a docx (unfortunately... I'm not a friend of this, but the project commands it).

My book looks like this:
```
book:
  chapters:
    - index.qmd
    - content/Chapter_01.qmd
    - content/Chapter_02.qmd
```

File `index.qmd` is empty, I would get an error during rendering if I would remove it. However, with file `index.qmd` comes automagically an additional chapter that is unwanted and in my opinion not useful at all.

Any idea how to unset the additional chapter?

Help is appreciated!

**Comment by dragonstyle** [comment]
I created an issue to track this as right now there isn't a great way to workaround this limitation. We'll get that issue fixed ASAP!

https://github.com/quarto-dev/quarto-cli/issues/2516

**Reply by iusgit** [reply]
Thank you, @dragonstyle  ! That was really fast.

I tested this modification with quarto dev 1.2.140 and indeed index.qmd is not needed in a pure docx.

I have however some observations: 
- when not using index.qmd, file paths (e.g. images) are now relative to the root path of the project and not  relative to the chapter's file path (if qmd files reside in an own content folder like shown above)
- At the end of the render process I see the message: pandoc: `content/quarto_test.docx: openBinaryFile: does not exist (No such file or directory)`. Again the file path of the docx get's confused.
- Probably I was a bit misleading above... I produce also non docx documents, so this shouldn't be prohibitive for the other documents; if this is not possible without major changes then it seems better to me accept that extra chapter...

Again thank you and sorry for the inconvinience I pose..

**Reply by dragonstyle** [reply]
Would it be possible for you to share a a test project I could use to take a closer look? I'm not seeing these errors when testing locally, but I likely don't have quite the right configuration to expose them....

**Reply by iusgit** [reply]
Hi,

find attached a test case. Hope it fails, too, for you...

[test.tar.gz](https://github.com/quarto-dev/quarto-cli/files/9618257/test.tar.gz)

**Reply by dragonstyle** [reply]
Thank you for sharing the test case - I was able to reproduce the issue with this test case! (The issue turned out to be the fact that the chapters were in a subdirectory). I have made another fix, which is available in v1.2.143 here https://quarto.org/docs/download/prerelease.html.

**Reply by iusgit** [reply]
So I tested the new version and indeed the docx is built without needing an index.qmd file.

The docx document is created in path `_book/content/quarto_test.docx` instead of `_book`, that is different to the stable version.

**Reply by dragonstyle** [reply]
Yeah, this is continuing to have additional ramifications that are making me feel bad about the current fix. We're going to revert the fix and continue to require an `index.qmd` file for the time being. I will transfer this to an issue and target it - we plan to do some additional work on books and will try to tackle this at the same time.

**Reply by kernie** [reply]
I agree. Thank you for your help!

**Comment by danieltomasz** [comment]
Hi I am putting a quick hacks for anyone who will have this problem 

I put
```
\let\standardclearpage\clearpage
\let\clearpage\relax 
```
otherwise leaving it empty,
then I put 
```
\let\clearpage\standardclearpage
```
where my actual text starts
I am sure that this can be done with pandoc template and conditional to 

but this is not solving the problem,, only hiding it
